modules/perinatal/controller/carnet-perinatal.events.ts

```diff
@@ -0,0 +1,75 @@
+import { CarnetPerinatalCtr } from './../carnet-perinatal.routes';
+import { EventCore } from '@andes/event-bus';
+import { userScheduler } from '../../../config.private';
+import moment = require('moment');
+let dataLog: any = new Object(userScheduler);
+dataLog.body = { _id: null };
+dataLog.method = null;
+
+// Al validar la prestacion de vacunacion, inserta la fecha de vacunacion en la inscripcion
+EventCore.on('perinatal:control:validacion', async ({ prestacion }) => {
```

Comment:
Si proviene de un dispatch de un elementoRUP, viene el registro como argumento. Te ahorras de buscarlo a mano y de tener el conceptId hardcoreado. 

```suggestion
EventCore.on('perinatal:control:validacion', async ({ prestacion, registro }) => {
```

Reply:
corregido